### ISSUE TITLE
InputGroup doesn't work

### ISSUE THREAD

**Issue by davidrhoderick** [react-bootstrap/react-bootstrap#6055]
## Describe the bug

When I try to use `<InputGroup>`, I get the following error:

```
Uncaught (in promise) Error: Invalid type passed to createElement(): [object Object]
```

## To Reproduce

Here's my code snippet:

```
import InputGroup from 'react-bootstrap/InputGroup'

...

<InputGroup>
  <Form.Control
    type="text"
    placeholder="Discount or Gift Card code"
    value={discountCode}
    onChange={event => dispatch(setDiscountCode(event.target.value))}
    onKeyDown={(event) => event.keyCode === 13 && dispatch(changeDiscountsStatus('idle'))}
  />
  <InputGroup.Append>
    <Button
      type="button"
      className="text-white"
      variant={discountCode.length === 0 ? "secondary" : "template3"}
      disabled={discountCode.length === 0}
      onClick={() => dispatch(changeDiscountsStatus('idle'))}
    >
      Apply
    </Button>
  </InputGroup.Append>
</InputGroup>
```

## Expected behavior

All the other components I'm using, `Accordion`, `Card`, `Button`, `Table`, `Form`, etc. work fine just by importing them and using them.

## Environment (please complete the following information)

- Operating System: Pop!_OS 21.04
- Browser, Version Firefox 92.0
- React-Bootstrap Version 1.6.3

## Additional context

I am using Preact but it seems to work with all other components.

**Comment by nishant3721** [comment]
@davidrhoderick  Can I work on it?

**Comment by kyletsang** [comment]
Seems to work in a regular CRA environment.  Can you create a minimal repro using preact?

**Comment by davidrhoderick** [comment]
Oof that was a doozy to set up; Codepen and Codesandbox just wouldn't work.  I set up a repo at https://github.com/davidrhoderick/preact-bootstrap-inputgroup.  If you comment out the `InputGroup` components, it'll work, but if you leave them in, you get the `[Object object]` error.

**Comment by kyletsang** [comment]
Thanks for the repro, it was very helpful.

I will push a fix later today.

**Comment by kyletsang** [comment]
Fixed in 1.6.4
